### PR TITLE
Fix URL Error

### PR DESCRIPTION
**Pull request by Xunzhuo**
Resolves:  #26689
Change /zh/docs/user-guide/load-balancer to /zh/docs/tasks/access-application-cluster/create-external-load-balancer/

### CODE REVIEW

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this:

```
./scripts/lsync.sh content/zh/docs/tutorials/services/source-ip.md
```

File: content/zh/docs/tutorials/services/source-ip.md [hunk 1]
```diff
@@ -395,5 +395,5 @@ $ kubectl delete deployment source-ip-app
 
 
 * 学习更多关于 [通过 services 连接应用](/zh/docs/concepts/services-networking/connect-applications-service/)
-* 学习更多关于 [负载均衡](/zh/docs/user-guide/load-balancer)
+* 学习更多关于 [负载均衡](/zh/docs/tasks/access-application-cluster/create-external-load-balancer/)
```
Comment (tengqm) [inline content/zh/docs/tutorials/services/source-ip.md:398]: 这里的learn more about ... 可翻译为”进一步学习..."，“学习更多关于..."这种不是中文表达方式。
另外，链接中文字如果是中文，不必在前后加空格。

Comment (Xunzhuo) [reply content/zh/docs/tutorials/services/source-ip.md:398]: 明白了，谢谢前辈🤝

Comment (Xunzhuo) [reply content/zh/docs/tutorials/services/source-ip.md]: 已将 “学习更多关于” 更改为 “进一步学习”😀

**Comment by tengqm** [comment]
@Xunzhuo Please perform a "complete" update to this page. There are other changes not synced in this PR.

**Pull request closed without merging**